taccsite_cms/static/site_cms/css/src/site.css

```diff
@@ -4,10 +4,12 @@
 /* SEE: https://confluence.tacc.utexas.edu/x/IAA9Cw */
 
 /* SETTINGS */
+/* Shared with Portal */
 @import url("../../../site_shared/css/src/_imports/settings/border.css");
 @import url("../../../site_shared/css/src/_imports/settings/color.css");
-@import url("../../../site_shared/css/src/_imports/settings/font.css");
+/* Unique to CMS */
 @import url("_imports/settings/props.space.css");
+@import url("_imports/settings/font.css");
```

Comment:
**Extra Change**
I am slowly, gradually deprecating the `site_shared` directory. It was an unnecessary placeholder for a "Portal-Core settings sync" problem I don't have yet, and should solve the right way before I have the problem.